src/components/Extensibility/ExtensibilityList.js

```diff
@@ -152,7 +183,16 @@ const ExtensibilityList = ({ overrideResMetadata, ...props }) => {
     >
       <DataSourcesContextProvider dataSources={resMetaData?.dataSources || {}}>
         <ExtensibilityErrBoundary key={urlPath}>
-          <ExtensibilityListCore resMetaData={resMetaData} {...props} />
+          {isExtensibilityCustomComponentsEnabled && resMetaData.customHtml ? (
+            <>
+              <div
+                dangerouslySetInnerHTML={{ __html: resMetaData.customHtml }}
+              ></div>
```

Comment:
```suggestion
              <div
                dangerouslySetInnerHTML={{ __html: resMetaData.customHtml }}
              />
```

Reply:
We already had this discussion 😄 
https://github.com/kyma-project/busola/pull/3471

Reply:
Ah yeah, goldfish memory :D